libraries/docker_container.rb

```diff
@@ -28,7 +28,7 @@ class DockerContainer < DockerBase
     property :extra_hosts, [Array, nil], coerce: proc { |v| Array(v).empty? ? nil : Array(v) }
     property :exposed_ports, PartialHashType, default: {}
     property :force, [true, false], default: false, desired_state: false
-    property :health_check, Hash, default: {}
+    property :health_check, [Hash, nil]
```

Comment:
```suggestion
    property :health_check, Hash
```

I'm fairly sure this is the same thing.